src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -15,6 +15,51 @@ namespace Microsoft.CodeAnalysis.CSharp
 {
     internal partial class Binder
     {
+        internal struct NamespaceOrTypeOrAliasSymbolWithAnnotations
+        {
+            internal static NamespaceOrTypeOrAliasSymbolWithAnnotations CreateNonNull(bool nonNullTypes, Symbol symbol)
+            {
+                if (symbol is null)
```

Comment:
>symbol [](start = 20, length = 6)

Do we hit the case where `symbol` is `null`? If not, maybe make this an assert instead.

Reply:
`BindTypeOrAliasOrKeyword` can return `null`.

---
In reply to: [202119879](https://github.com/dotnet/roslyn/pull/28484#discussion_r202119879) [](ancestors = 202119879)